src/jira_macro.rb

```diff
@@ -233,9 +244,17 @@ def get_credentials(parent)
   end
 
   def parse_fields(attrs)
-    fields_param = attrs['fields'] 
-    fields = fields_param ? fields_param.split(',').map(&:strip) : nil
-    fields || ['key', 'summary', 'status']
+    raw = attrs['fields']
+    return ['key', 'summary', 'status'] if raw.nil? || raw.strip.empty?
+
+    # Allow single-quoted field names (including commas) by converting them to standard CSV double quotes.
+    # Supports escaping single quotes inside a single-quoted token by doubling them: 'Another ''Quoted'' Field'
+    normalized = raw.gsub(/'([^']*(?:''[^']*)*)'/) do
```

Comment:
Let#s actually check what asciidoc does. I think it might just not work and we need to escape differently.